### DISCUSSION TITLE
Where is org.eclipse.osgi.services package stored

### DISCUSSION THREAD

**Discussion by pintify** [eclipse-kura/kura#5110] [Q&A]
I'm trying to upgrade the dependencies on my developments but I seem to fail finding the source of the last org.eclipse-osgi.services. In maven I've only seen up to version 3.2 or 3.5 (web2) but I've been unable to find a repository with the 3.9 Kura is using. I only managed to reach [this](https://download.eclipse.org/staging/2020-12/reports/download.eclipse.org/staging/2020-12/index/org.eclipse.osgi.services_3.9.0.v20200511-1725.html) documentation with the official distribution of the jar, but no maven repository anywhere. Could you provide me some help?

**Comment by mattdibi** [comment]
A starting point might here: https://github.com/eclipse/kura/blob/588c0a237e4a47344d413691e5086fd4452397c8/target-platform/p2-repo-equinox_3.16.0/pom.xml#L15

We're downloading all things related to OSGi in our target platform here: target-platform/p2-repo-equinox_3.16.0/pom.xml

Maybe take a look at that if it fits your needs. Hope this helps.

**Reply by pintify** [reply]
Hum... Ok, now I see how Kura loads it, should I need to load the whole equinox platform to access that package then? Or is there somewhere easier to get it from which makes a ligther build?

**Reply by pintify** [reply]
Following the thread through equinox I found [this](https://mvnrepository.com/artifact/org.eclipse.platform/org.eclipse.osgi.services) in maven repository and also the destination for the equinox [build itself](https://repo.eclipse.org/content/repositories/eclipse/org/eclipse/platform/), but in the last, all version prior to 3.10 are not present for some reason. I'll give the Central repo version a try. Thanks!

**Comment by mattdibi** [comment]
Feel free to reopen this if the need may arise
